content/remote-development/init-command.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Init Command
+menu:
+  main:
+    parent: 'remote-development'
+    weight: 90
+
+weight: 90
+---
+## Using the Init Command
+
+```
+cp-remote init [token]
+```
+
+The `init` command will initialise a remote environment using a token generated by the Continuous Pipe UI. 
```

Comment:
For consistency across the docs, please use "ContinuousPipe console" and add a link i.e [ContinuousPipe console](https://ui.continuouspipe.io/.